pkg/Library/UX/UXNodeTypes.js

```diff
@@ -31,5 +31,10 @@ export const UXNodeTypes = {
     category,
     description: 'Collects UX action definitions for buttons and controls',
     type: `$library/UX/Nodes/UXActionSetNode`
+  },
+  UXSelectionList: {
+    category,
+    description: 'Render a list, allow user to select items',
```

Comment:
nit - i think all (most) rest of the descriptions are in 3rd person "Renders a list..."; or if you prefer it this way, i can update the rest of them to match

Reply:
No preference, my mistake.

Reply:
changed it to `Renders`